Source/AliveLibAO/PauseMenu.cpp

```diff
@@ -147,7 +147,7 @@ ALIVE_VAR(1, 0x5080C6, saveName, saveNameBuffer_5080C6, {});
 
 const char *gLevelNames_4CE1D4[20] =
 {
-    "�",
+    "�",
```

Comment:
github is displaying this in a strange way, wasn't it just a period previously?

Reply:
Not sure what happened here, will fix.

Reply:
This was not a period, it was `¸`. Do we want to keep it like that?
https://unichar.app/web/char/?category=Punctuation&char=%C2%B8

Reply:
It looks like it should just be a blank string "" looks like a reversing mistake/bug